libs/tree-sitter-wing/grammar.js

```diff
@@ -515,7 +516,44 @@ module.exports = grammar({
     await_expression: ($) => prec.right(seq("await", $.expression)),
     defer_expression: ($) => prec.right(seq("defer", $.expression)),
     parenthesized_expression: ($) => seq("(", $.expression, ")"),
+    
+    json_literal: ($) => choice(
+      seq(field("type", choice($.json_container_type)), field("element", $._json_value)),
+    ),
+
+    json_container_type: ($) => choice(
+      "Json",
+      "MutJson"
+    ),
+
+    _json_value: ($) => choice(
```

Comment:
I don't think we need a `_json_value`. It seems like any place it's used is basically a place where we can have some expression that evaluates to a json type. So is json value just an `$.expression`.

You might be worried that if you define some `json { a: function_call(1) }` it'll be fine as long as `function_call(1)` returns a json type but it'll fail if it evaluates to a non-json type. In that case it's the **type-checker**'s job to give an error not the grammar/parser. Perhaps this is a source of confusion, or perhaps I don't understand something here.